test/Component/Configuration/Loader/TypescriptLoaderTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+namespace Hostnet\Component\Webpack\Configuration\Loader;
+
+use Hostnet\Component\Webpack\Configuration\CodeBlock;
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+
+/**
+ * @covers Hostnet\Component\Webpack\Configuration\Loader\TypescriptLoader
+ */
+class TypescriptLoaderTest extends \PHPUnit_Framework_TestCase
+{
+    public function testConfigTreeBuilder()
+    {
+        $tree = new TreeBuilder();
+        $node = $tree->root('typescript')->children();
```

Comment:
Can you also add the configuration option in the functional tests so we know it works? Oh and don't forget the readme